rpm-build/SPECS/securedrop-workstation-dom0-config.spec

```diff
@@ -30,33 +30,44 @@ configuration over time.
 %{__python3} setup.py build
 
 %install
-%{__python3} setup.py install --skip-build --root %{buildroot}
+%{__python3} setup.py install --no-compile --skip-build --root %{buildroot}
+install -m 755 -d %{buildroot}/opt/securedrop/launcher
+install -m 755 -d %{buildroot}/opt/securedrop/launcher/sdw_updater_gui
 install -m 755 -d %{buildroot}/srv
 install -m 755 -d %{buildroot}/srv/salt/sd
 install -m 755 -d %{buildroot}/srv/salt/sd/sd-app
+install -m 755 -d %{buildroot}/srv/salt/sd/sd-proxy
```

Comment:
do we not need sd-whonix folder also (and in MANIFEST.in)? we have a Jinja template for the v3 auth creds there